{
  "uid" : "88fd832e3747dd7d",
  "name" : "test_02_login[-123456-请填写用户名和密码]",
  "fullName" : "test.test_web.test_login.Test_Login#test_02_login",
  "historyId" : "7c7344f1741d74967ee6bb173127859c",
  "time" : {
    "start" : 1668966956973,
    "stop" : 1668966961150,
    "duration" : 4177
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert 0",
  "statusTrace" : "self = <test.test_web.test_login.Test_Login object at 0x000001E2D2C77070>\nusername = '', password = 123456, expected = '请填写用户名和密码'\n\n    @pytest.mark.parametrize(\"username, password, expected\",\n                             testcase[\"c_example\"])\n    def test_02_login(self, username, password, expected):\n        self.home_proxy.to_login()\n        time.sleep(2)\n        self.login_proxy.enter(username, password)\n        time.sleep(2)\n        loginErr_text = self.login_proxy.get_loginErr()\n        assert loginErr_text == expected\n>       assert 0\nE       assert 0\n\ntest\\test_web\\test_login.py:51: AssertionError",
  "flaky" : false,
  "beforeStages" : [ {
    "name" : "_xunit_setup_method_fixture_Test_Login",
    "time" : {
      "start" : 1668966955769,
      "stop" : 1668966956972,
      "duration" : 1203
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert 0",
    "statusTrace" : "self = <test.test_web.test_login.Test_Login object at 0x000001E2D2C77070>\nusername = '', password = 123456, expected = '请填写用户名和密码'\n\n    @pytest.mark.parametrize(\"username, password, expected\",\n                             testcase[\"c_example\"])\n    def test_02_login(self, username, password, expected):\n        self.home_proxy.to_login()\n        time.sleep(2)\n        self.login_proxy.enter(username, password)\n        time.sleep(2)\n        loginErr_text = self.login_proxy.get_loginErr()\n        assert loginErr_text == expected\n>       assert 0\nE       assert 0\n\ntest\\test_web\\test_login.py:51: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "a8bbf87be86d2791",
      "name" : "stdout",
      "source" : "a8bbf87be86d2791.txt",
      "type" : "text/plain",
      "size" : 78
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true,
    "stepsCount" : 0
  },
  "afterStages" : [ {
    "name" : "_xunit_setup_method_fixture_Test_Login::0",
    "time" : {
      "start" : 1668966961154,
      "stop" : 1668966963188,
      "duration" : 2034
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "team('小马团队')"
  }, {
    "name" : "parentSuite",
    "value" : "test.test_web"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "Test_Login"
  }, {
    "name" : "host",
    "value" : "xiaoma"
  }, {
    "name" : "thread",
    "value" : "27768-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_web.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "expected",
    "value" : "'请填写用户名和密码'"
  }, {
    "name" : "password",
    "value" : "123456"
  }, {
    "name" : "username",
    "value" : "''"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "team('小马团队')" ]
  },
  "source" : "88fd832e3747dd7d.json",
  "parameterValues" : [ "'请填写用户名和密码'", "123456", "''" ]
}